libs/dtm/dtm_recovery.py

```diff
@@ -159,3 +161,25 @@ def process_restart(self, master_node, pod_prefix, container_prefix, process,
         time.sleep(recover_time)
         # TODO : Check if process has started
         self.log.info("Process restarted ")
+
+    def perform_copy_objects(self, workload, que):
+        """
+        function to perform copy object for dtm test case in background
+        :param workload: Python dict containing source and destination bucket and object
+        :param que: Multiprocessing Queue to be used for returning values (Boolean,dict)
+        """
+        results = list()
+        for obj_name in workload["obj_list"]:
+            resp = self.s3_obj.copy_object(source_bucket=workload["source_bucket"],
+                                           source_object=obj_name,
+                                           dest_bucket=workload["dest_bucket"],
+                                           dest_object=obj_name)
+            if resp[0]:
+                results.append(True)
+            else:
+                results.append(False)
+                break
+        if all(results):
+            que.put([True])
+        else:
+            que.put([False, "Copy Object operation failed"])
```

Comment:
Please add object name for which the copy object failed.

Reply:
added